src/Microsoft.Azure.WebJobs.Extensions.Kafka/Output/KafkaAttribute.cs

```diff
@@ -15,6 +15,14 @@ namespace Microsoft.Azure.WebJobs.Extensions.Kafka
     [Binding]
     public sealed class KafkaAttribute : Attribute
     {
+        private int? compressionLevel;
+        private int? maxMessageBytes;
```

Comment:
Wonder if we shouldn't change this to have the default values set on the field/property declarations instead of having in the getter implementation body?

Reply:
I guess it looked a bit weird the way I did it, but my reason was to start a pattern where it could be determined whether a setting had been specified or not. I relates to the function/global setting scope issue.

Reply:
I understand, but I haven't see any usage. Are we this information (whether or not a setting was set)?

Reply:
No, I didn't get that far. I pushed a commit, to demonstrate.